api/app/controllers/spree/api/v2/platform/images_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class ImagesController < ResourceController
+          before_action -> { doorkeeper_authorize! :write, :admin }, only: WRITE_ACTIONS << :upload
+
+          def upload
+            url_helpers = Rails.application.routes.url_helpers
+
+            blob = ActiveStorage::Blob.create_after_upload!(
+              io: params[:file],
+              filename: params[:file].original_filename,
+              content_type: params[:file].content_type
+            )
+            render json: { location: url_helpers.rails_blob_url(blob, only_path: true) }, content_type: 'text / html'
```

Comment:
Why do you need to change the `content_type` here?

Is `'text / html'` the correct way to specify this MIME type? Shouldn't it be `'text/html'` (no spaces)

Maybe the best way to ask Rails to give us the correct MIME type:
https://api.rubyonrails.org/classes/Mime/Type.html

```rb
Mime::Type.lookup('text/html')
```